src/components/swapv2/LimitOrder/ListOrder/SummaryNotify.tsx

```diff
@@ -40,7 +40,7 @@ export default function SummaryNotify({
         {formatAmountOrder(takingAmount, takerAssetDecimals)} {takerAssetSymbol}
       </Text>{' '}
       <Text as="span" color={theme.subText}>
-        at a {takerAssetSymbol} price of {rate} {makerAssetSymbol}
+        at {takerAssetSymbol} price of {rate} {makerAssetSymbol}
```

Comment:
should wrap with  <Trans>

Reply:
Trans wrapper
<img width="523" alt="image" src="https://user-images.githubusercontent.com/33005392/211707361-51f653ad-c957-4fdd-a102-2e346a2aaa11.png">